File: java/Ram.java
public class Ram extends Komponen {
    private int kapasitasGB;
    private String ddr;

    public Ram(String merk, String nama, int kapasitasGB, String ddr) {
        super(merk, nama);
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
    }

    public int getKapasitasGB() {
        return kapasitasGB;
    }

    public String getDdr() {
        return ddr;
    }

    public void setKapasitasGB(int kapasitasGB) {
        this.kapasitasGB = kapasitasGB;
    }

    public void setDdr(String ddr) {
        this.ddr = ddr;
    }

    @Override
    public String toString() {
        return super.toString() + " | " + kapasitasGB + " GB " + ddr;
    }
}

File: py/ram.py
from komponen import Komponen

class Ram(Komponen):
    def __init__(self, merk: str, nama: str, kapasitas_gb: int, ddr: str):
        super().__init__(merk, nama)
        self.kapasitas_gb = kapasitas_gb
        self.ddr = ddr

    def get_kapasitas_gb(self):
        return self.kapasitas_gb

    def get_ddr(self):
        return self.ddr
